spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubAutoConfiguration.java

```diff
@@ -120,9 +121,14 @@ public ExecutorProvider subscriberExecutorProvider() {
 
 	@Bean
 	@ConditionalOnMissingBean
-	public PubSubMessageConverter pubSubMessageConverter() {
-		return new JacksonPubSubMessageConverter(
-				this.gcpPubSubProperties.getTrustedPackages());
+	public ObjectMapper objectMapper() {
```

Comment:
Sorry, but seems for me we talked with you that no an explicit `ObjectMapper` bean.
Everything should be deferred to end-user decision.